frontend/public/components/monitoring/alerting.tsx

```diff
@@ -102,26 +103,29 @@ const pollerTimeouts = {};
 
 const silenceAlert = (alert: Alert) => ({
   callback: () => history.replace(`${SilenceResource.plural}/~new?${labelsToParams(alert.labels)}`),
-  label: 'Silence Alert',
+  label: i18next.t('monitoring~Silence alert'),
```

Comment:
I believe the changes here are covered in #7386 :/ (Sorry for the duplicate effort.)

Reply:
Yes. it is covered in #7386.